doc/conf.py

```diff
@@ -0,0 +1,60 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+import pkg_resources
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Everviz'
+copyright = '2020, Equinor'
+author = 'Equinor'
+
+def get_version():
+    # Retrieve the everviz installed version.
+    everviz_version = pkg_resources.get_distribution(project).version
+    print("everest version: {}".format(everviz_version))
+    return everviz_version
+
+
+# The short X.Y version
+version = get_version()
+# The full version, including alpha/beta/rc tags
+release = version
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'classic'
```

Comment:
Can I suggest a more modern theme:
```suggestion
html_theme = "sphinx_rtd_theme"
```